pom.xml

```diff
@@ -6,19 +6,19 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-build</artifactId>
-		<version>2.1.0.BUILD-SNAPSHOT</version>
+		<version>2.1.3.RELEASE</version>
 		<relativePath/>
 	</parent>
 
 	<artifactId>spring-cloud-stream-binder-kinesis-parent</artifactId>
-	<version>1.1.0.BUILD-SNAPSHOT</version>
+	<version>1.1.1.BUILD-SNAPSHOT</version>
```

Comment:
Well, that's not. Since it is about a new feature in the project, so we will go a new minor release. Therefore: `1.2.0.BUILD-SNAPSHOT`